### ISSUE TITLE
Addon module results in 500 internal server error

### ISSUE THREAD

**Issue by lucasRolff** [openprovider/OP-WHMCS7#33]
When trying to access the addon in WHMCS, it results in a 500 internal server error:

```
ReflectionException: Class \OpenProvider\WhmcsDomainAddon\Controllers\Admin\Supportcontroller does not exist in /path/modules/registrars/openprovider/vendor/wedevelopcoffee/wpower/src/Core/Launcher.php:40
Stack trace:
#0 /path/modules/registrars/openprovider/vendor/wedevelopcoffee/wpower/src/Core/Launcher.php(40): ReflectionClass->__construct('\\OpenProvider\\W...')
#1 /path/modules/registrars/openprovider/vendor/wedevelopcoffee/wpower/src/helpers.php(20): WeDevelopCoffee\wPower\Core\Launcher->launchClass('\\OpenProvider\\W...')
#2 /path/modules/registrars/openprovider/vendor/wedevelopcoffee/wpower/src/wPower.php(88): wLaunch('\\OpenProvider\\W...')
#3 /path/modules/registrars/openprovider/vendor/wedevelopcoffee/wpower/src/Controllers/Dispatcher.php(99): WeDevelopCoffee\wPower\wPower->launch('\\OpenProvider\\W...')
#4 /path/modules/addons/openprovider/openprovider.php(75): WeDevelopCoffee\wPower\Controllers\Dispatcher->dispatch('index', Array)
#5 /path/admin/addonmodules.php(0): openprovider_output(Array)
#6 {main}
```

**Comment by theemstra** [comment]
Can confirm, got the same issue.
WHMCS 7.7.1

**Comment by willdashwood** [comment]
Duplicate of #30 

The addon module has been added, then removed, then add back in again. I can only assume in error as I don't think it is intended to be part of this module.

**Comment by lucasRolff** [comment]
This issue is not a duplicate of #30 - it's two different things.

**Comment by willdashwood** [comment]
Sorry I should have clarified. What I meant was that issues with that addon module are unlikely to be addressed here as it appears to be a mistake that it's included in the first place. Of course that could have all changed. I've never tried to get it working, not even sure what it does.

**Comment by lucasRolff** [comment]
I've fixed the error in PR #36

**Comment by soberoctober** [comment]
historically, the add-on has been a test polygon for all things experimental.
in its current role the add-on module serves as an export log utility(activity and module logs).
a working version.

the following update of the module will include a working prototype.
